docs/pages/includes/database-access/mongodb-required-database-access.mdx

```diff
@@ -0,0 +1,8 @@
+<Admonition type="note">
```

Comment:
I don't think this needs to be an `Admonition`. There is more text inside an `Admonition` at the end of this H2 section than outside, so this `Admonition` stands out less than it could.

Also, I think we can make it more explicit that these instructions apply only if a user decides to use a non-wildcard `--db-names` value, and that the `tctl users add` command above already allows access to the `admin` database.

Reply:
I've updated it. Can you take a look? Thanks.